src/cmd/flux-job.c

```diff
@@ -1471,12 +1480,27 @@ int cmd_list_ids (optparse_t *p, int argc, char **argv)
     if (!(h = flux_open (NULL, 0)))
         log_err_exit ("flux_open");
 
+    /* if no job state specified by user, pick first job state of
+     * depend, which means will return immediately
```

Comment:
Would it be more correct to say "will return as soon as the job-list module is aware of the job" instead of "immediately"?